edi_oca/models/edi_exchange_record.py

```diff
@@ -603,3 +612,28 @@ def delayable(self, **kw):
 
     def _job_retry_params(self):
         return {}
+
+    def _compute_is_obsolete(self):
```

Comment:
What if we do this check immediately at creation?
Right after creation we can look for older duplicates rather than new and set their state as `obsolete` after commit.